config.json

```diff
@@ -141,7 +141,9 @@
       "slug": "nucleotide-count",
       "uuid": "d72cf2c3-9f30-4df1-a69b-04c638bb2426",
       "core": false,
-      "unlocked_by": null,
+      "unlocked_by": [
```

Comment:
Actually, these unlocked by sections are supposed to be strings, not arrays.

Reply:
really? configlet hasn't been complaining

Reply:
I see you're right. let's not make this change here lol. I'm adding problems in other pull requests and I think it could lead to conflicts